sklearn/metrics/scorer.py

```diff
@@ -294,7 +294,7 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
     >>> ftwo_scorer = make_scorer(fbeta_score, beta=2)
     >>> ftwo_scorer
     make_scorer(fbeta_score, beta=2)
-    >>> from sklearn.grid_search import GridSearchCV
+    >>> from sklearn.model_selection.search import GridSearchCV
```

Comment:
> Remember that every time you add such a note we get an e-mail :)

The internet is buckling under the weight of those emails!

On 7 June 2015 at 02:26, Raghav R V <notifications@github.com> wrote:

> In sklearn/metrics/scorer.py
> <https://github.com/scikit-learn/scikit-learn/pull/4294#discussion_r31869363>
> :
>
> > @@ -294,7 +294,7 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
> >      >>> ftwo_scorer = make_scorer(fbeta_score, beta=2)
> >      >>> ftwo_scorer
> >      make_scorer(fbeta_score, beta=2)
> > -    >>> from sklearn.grid_search import GridSearchCV
> > +    >>> from sklearn.model_selection.search import GridSearchCV
>
> Oops sorry :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/4294/files#r31869363>.
>